stan/math/rev/core/var.hpp

```diff
@@ -7,6 +7,9 @@
 #include <boost/math/tools/config.hpp>
 #include <ostream>
 #include <vector>
+#include <complex>
+#include <cassert>
+#include <limits>
```

Comment:
I didn't see anywhere `<limits>` gets used.

You won't need `<cassert>` after changing the asserts to exceptions.